### DISCUSSION TITLE
vue dev-server proxying - only loads index.html // status 404

### DISCUSSION THREAD

**Discussion by poopato** [coder/code-server#2835] [Q&A]
Hey there! 
I have set up an Ubuntu 20.04 vServer with code-server 3.9.
Im using the authentication provided by the --link flag.

I want to access my live-preview devserver provided by vue.
When i access it with https://<domain>.cdr.co/absproxy/8101 it only loads the index.html which ends up as an empty white page. Same for /proxy/8101
In the console I get  status 404 for every GET request.

I dont really understand the approach provided by the docs.

I hope someone can help me :)

![image](https://user-images.githubusercontent.com/76976759/110320599-dbc6c100-8010-11eb-9630-0bfd9c33d6e3.png)

my vue.config.js file: 
`module.exports = {
    devServer: {
      host: '0.0.0.0',
      allowedHosts: [
        '.cdr.co',
        'localhost-poopato.cdr.co'
      ]
  }
  };`

**Comment by code-asher** [comment]
You might need to inform the dev server about the base path. For example `app.js` might be requesting `/app.js` instead of `/proxy/8101/app.js`.

If you're using Webpack I think changing `publicPath` should make it work.

**Reply by poopato** [reply]
Hey! Thanks for your reply. I have tried to publicPath approach
so I edited my config file to this: 
`module.exports = { devServer: { host: '0.0.0.0', allowedHosts: [ '.cdr.co', 'localhost-poopato.cdr.co' ] }, publicPath: 'https://localhost-poopato.cdr.co/proxy/9000/' };` I also tried '/proxy/9000/', '/proxy/9000',

Somehow it didnt wanted to work... Once I used '/absproxy/9000/' instead it did the trick...

Maybe i messed something else up... but it works now... so thanks :D

**Reply by code-asher** [reply]
Perfect! I should have used `absproxy` in my example; that's the correct one to use when an application needs a base path.
